pearcej/cppds
# Solved cross-reference errors in section 9.20 -- Dijkstra’s Algorithm

### Description
This pull request transfers the "Figures" to the  "Graphs" folders, aligning them with the formatting used in the RST code (this is why there are 91 files changed -- its the PNG's). Additionally, it fixes any references to these images throughout the document.

### Testing 
1. Check out this branch and build the project.
2. Locally view the project.
3. Navigate to section 9.20.
4. Confirm that the figures are displayed correctly.
5. Ensure that references to the images in the text are accurate and functional.

---
![image](https://github.com/pearcej/cppds/assets/97637517/b03a4083-a2bd-4da9-9417-0bce35a5cea3)

fix #328
